src/components/ChatGhostUI/ContentLoadingElements/ContentLoadingTwoLines/index.native.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ContentLoader, {Rect, Circle} from 'react-content-loader/native';
+
+const ContentLoadingTwoLines = () => (
+    <ContentLoader
+        height={80}
+        width="100%"
+        title=""
```

Comment:
Why is is empty prop used at all?

Reply:
If I don't set empty, it shows a text `Loading...` that comes default from `react-content-loader`

Reply:
ok